edgesql_kaggle.py

```diff
@@ -2,14 +2,14 @@
 import os
 import zipfile
 import json
+import requests
```

Comment:
Maybe, just a suggestion, since we will use only the `RequestExceptions` from requests module, we could;
```suggestion
from requests.exceptions import RequestException
```
What do you think?